src/lms-harmonizer/tests_integration_mssql/procs/mssql/harmonize_assignment_submissions/test_lmsassignment_submissions.py

```diff
@@ -78,8 +78,8 @@ def it_should_insert_the_submissions_successfully(
             insert_lms_section(connection, SIS_SECTION_ID, source_system)
             insert_edfi_section(connection, SIS_SECTION_ID)
             connection.execute(
-                """UPDATE LMS.LMSSECTION SET
-                    EdFiSectionId = (SELECT TOP 1 ID FROM EDFI.SECTION)"""
+                """update lms.lmssection set
+                    edfisectionid = (select id from edfi.section limit 1)"""
```

Comment:
Needs to go back to the old syntax with `top 1`. Same everywhere else in this SQL Server file.

Reply:
Yep, done by accident. Just reverted.